README.md

```diff
@@ -41,3 +41,161 @@ We have created a GitHub rule which allows major code changes to be merged into
 ## Team Ceremonies
 
 We have a weekly 30 minute video calls to go over the changes of roles within the team as well as maintain the board. In these 30 minutes sessions we also go over any aspects that needs to be clarified within the team or passed to the originator.
+
+# Application setup and navigation
+
+In this section we will walk you through how to setup and navigate through the New Bank Java application.
+
+## Setup Google Authentication
+
+Before initiating the NewBank app, you will need to install the Google Authentication Application on a mobile device (iOS or Android), when you have completed the installation follow these steps:
+
+1. Tap on `Get Started`
+2. Select `Enter a setup key`
+3. On the `Account` text field enter any custom name (e.g. MyName)
+4. On the `Key` text field enter this exact key: `NY4A5CPJZ46LXZCP`
+5. Make sure the account is set on `Time-based` from the drop down menu
+6. Tap on the `Add` button
+
+## Executing the NewBank App
+
+In order to run the NewBank Java application, you will first need to make sure you have at least Java SDK 11 installed in your designated IDE. You will then have to run the following commands:
+
+1. Open a terminal from the root of the New-Bank-Group-4 project
+2. In the terminal enter the `javac newbank/server/NewBankServer.java` command to compile the code
+3. Enter the `java newbank/server/NewBankServer.java` command on the same terminal
+4. Then you should get a message on the terminal stating that the New Bank Server is listening to a local port number. Leave this terminal open and move on to the next step
+5. Open a second terminal from the root of the New-Bank-Group-4 project
+6. Enter the `javac newbank/client/ExampleClient.java ` command
+7. Enter `java newbank/client/ExampleClient.java ` command on the same second terminal
```

Comment:
```suggestion
7. Enter `java newbank/client/ExampleClient` command on the same second terminal
```
It should be entered without the .java I think. I just tried it and it only worked when not entering .java. (google says so too, but I have no idea ^^)

Reply:
this only applies to line 70, not 69